PERL/ASGSUtil.pm

```diff
@@ -103,6 +103,27 @@ sub setParameter {
    }
    return;
 }
+
+# create a hash of properties from run.properties
+sub readProperties {
+   my ( $runProp_ref, $fileName ) = @_;
+   # open properties file
+   my $rp;
```

Comment:
don't think `my $rp;` is needed, you can use it inline with the `open` in the `if` or `unless`.

Reply:
including the `my` inline did not work for me, I got error messages about undeclared variables

Reply:
I will look at this again. ty